web/src/message_list_view.js

```diff
@@ -619,101 +649,119 @@ export class MessageListView {
         return undefined;
     }
 
-    build_message_groups(message_containers) {
-        const start_group = () => ({
-            message_containers: [],
-            message_group_id: _.uniqueId("message_group_"),
-        });
+    build_message_groups(messages) {
+        let group_message_containers = [];
```

Comment:
rename to `current_group_message_containers` probably, and move below `let current_group`